manual-tests/01 - Világ inicializáció sanity check.md

```diff
@@ -0,0 +1,27 @@
+# 01 - Világ inicializáció sanity check
+
+A világ inicializálás alapvető működésést ellenőrzi. Ha ez a teszt nem sikeres,
+akkor a többi inicializáló teszttel nem érdemes foglalkozni.
+
+## Bemeneti adatok
```

Comment:
Szerintem jobb lenne valahogy úgy struktúrálni  a fájlt, hogy ezek a bemeneti adatok a megfelelő helyen legyenek. Ebben a struktúrában kicsit össze-vissza kell olvasni a dolgokat.

Reply:
Egyébként meg lehet szokni, csak elsőre olvasva az az első kérdésem a bemeneti adatokra, hogy mik hova miért? De később kiderül.

Reply:
Véleményem szerint valóban nem egyértelmű elsőre, hogy milyen bemeneti adatokról van szó, viszont nagyobb mennyiség esetén tényleg érdemes külön szervezni ezeket, az egységesség érdekében pedig a többi esetben is jobb így tenni. Ezért én azt javasolnám, hogy ahol a lépések leírásánál a bemeneti adatokra van hivatkozva, ott lehetne egy megjegyzés (pl.: lásd fent-lent-akárhol), hogy hol kellene azokat keresni, így egyértelműbb lenne, de megmaradna az olvashatósága is.